File: Python Code/Daily_Flash_Code_0/Week10/25Mar/56-DailyFlash_Solutions/25-MarSolution_Three/C/prog1.c
#include<stdio.h>
#include<math.h>
int main() {
	int l;
	float sum = 0;
	int x, y;
	printf("X, Y : ");
	scanf("%d %d", &x, &y);
	printf("Length : ");
	scanf("%d", &l);
	for(int i = 1; i <= l; i++) {
		sum += i / pow((x + y), i);
	}
	printf("%f", sum);
}

File: Python Code/Daily_Flash_Code_0/Week10/25Mar/56-DailyFlash_Solutions/25-MarSolution_Three/Java/prog1.java
import java.util.Scanner;
class Demo {
	public static void main(String[] args) {
		float x, y, l, mul = 0;
		Scanner sc = new Scanner(System.in);
		l = sc.nextFloat();
		x = sc.nextFloat();
		y = sc.nextFloat();
		for(int i = 1; i <= l; i++) {
			mul += (i) / Math.pow(x + y, i);
		}
		System.out.println(mul);
	}
}
